File: praktikum_algopascal/C/procedure_2.c
//procedure 2 tentang sisi persegi

#include <stdio.h>

void luas(float sisi)
{
	float luas;
	luas = sisi * sisi;
	printf("Luas Persegi: %2.f", luas);
}

int main()
{
	float s;

	printf("Masukan Sisi Persegi: ");
	scanf("%f", &s);

	luas(s);

	return 0;
}

File: praktikum_algopascal/pascal/procedure_2.pas
//prosedur 2 tentang luas persegi

program prosedur2;
	uses crt;
		var
			s: real;
	procedure luasPersegi(sisi: real);
	var luas : real;
						
begin
	luas := s * s;
	writeln('Luas Persegi: ', luas:2:0);
end;

begin
	clrscr;
	write('Masukan Sisi Persegi: ');
	readln(s);
	luasPersegi(s);
	readln;
end.